src/js/components/Drop/DropContainer.js

```diff
@@ -109,8 +109,15 @@ const DropContainer = forwardRef(
           if (attr !== null) clickedPortalId = parseInt(attr, 10);
           node = node.parentNode;
         }
+        // Check if the click happened within the dropTarget
+        const clickInsideDropTarget =
+          (dropTarget.current && dropTarget.current.contains(event.target)) ||
```

Comment:
```suggestion
          (dropTarget?.current && dropTarget.current.contains(event.target)) ||
```
Do we need the `?` here? I saw we had it in a previous commit so wasn't sure

Reply:
yes good catch